### ISSUE TITLE
Conflict with dfunckt/django-rules

### ISSUE THREAD

**Issue by IrishGussy** [arteria/django-background-tasks#34]
I am using django-rules (https://github.com/dfunckt/django-rules) which requires my INSTALLED_APPS to include 'rules.apps.AutodiscoverRulesConfig'.

This does not appear to be a module but is a method that auto-discovers modules as follows -

```
class AutodiscoverRulesConfig(RulesConfig):
    def ready(self):
        from django.utils.module_loading import autodiscover_modules
        autodiscover_modules('rules')
```

When I use this in conjunction with `manage.py process_tasks`, I get the message `ImportError: No module named AutodiscoverRulesConfig`. I can rectify the problem by changing DBTaskRunner.autodiscover() as follows -

```
for app in settings.INSTALLED_APPS:
    if app <> 'rules.apps.AutodiscoverRulesConfig':
        try:
            app_path = import_module(app).__path__
        except AttributeError:
            continue
        try:
            imp.find_module('tasks', app_path)
        except ImportError:
            continue

        import_module("%s.tasks" % app)
```

This works fine because rules are not used for background processes but I have difficulty deploying. Any thoughts on how I should proceed?

**Comment by simonv3** [comment]
I have a similar error but with with a config I define myself, eg: 

```python
from django.apps import AppConfig

class DashboardConfig(AppConfig):
    name = 'dashboard'
    verbose_name = 'Dashboard'

    def ready(self):
        import dashboard.signals.handlers
```

Which then gets added in `INSTALLED_APPS` in settings: 

```
'dashboard.apps.DashboardConfig',
```

This basically makes sure that signals are registered properly. 

The stacktrace:

```python
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 2218, in _find_and_load_unlocked
AttributeError: 'module' object has no attribute '__path__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/background_task/management/commands/process_tasks.py", line 76, in handle
    autodiscover()
  File "/home/ukloportal/.virtualenvs/uklo/lib/python3.4/site-packages/background_task/tasks.py", line 265, in autodiscover
    app_path = import_module(app).__path__
  File "/usr/local/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2221, in _find_and_load_unlocked
ImportError: No module named 'dashboard.apps.DashboardConfig'; 'dashboard.apps' is not a package
```

**Comment by walterrenner** [comment]
@IrishGussy @simonv3 is this still an issue?

I think this was fixed with 090140700ce469157dce39702cc5d27b31b08856 in v1.0.15

**Comment by philippeowagner** [comment]
Closing this ... please reopen in case this still persists.
